frontend/src/metabase/ui/components/inputs/Select/Select.styled.tsx

```diff
@@ -126,17 +129,17 @@ export const getSelectItemsOverrides = (
       padding: "0.75rem",
     },
     item: {
-      color: theme.fn.themeColor("text-dark"),
+      color: "var(--mb-color-text-primary)",
       fontSize: getItemFontSize(size),
       lineHeight: getItemLineHeight(size),
       padding: theme.spacing.sm,
       "&[data-hovered]": {
-        color: theme.fn.themeColor("brand"),
-        backgroundColor: theme.fn.themeColor("brand-lighter"),
+        color: "var(--mb-color-text-selected)",
+        backgroundColor: "var(--mb-color-background-selected)",
       },
       "&[data-selected]": {
         color: theme.fn.themeColor("text-white"),
-        backgroundColor: theme.fn.themeColor("brand"),
+        backgroundColor: "var(--mb-color-background-brand)",
       },
```

Comment:
I think you've addressed this in the previous PR, but note that we are using `--mb-color-text-selected` and `--mb-color-background-selected` for the "hovered" state in both cases, for the Select and Popover. It makes me wonder if we should add a separate semantic color for this.

Reply:
I was having the same thought at some point as well. Maybe we need more semantic colors like `hovered`. Let's do that in a follow up PR!